<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Zoomer</class>
 <widget class="QFrame" name="Zoomer">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>542</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Zoom</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::WinPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Sunken</enum>
  </property>
  <widget class="QGraphicsViewCustom" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>242</width>
     <height>482</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="x_label">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>0</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>x:</string>
   </property>
  </widget>
  <widget class="QLabel" name="y_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>0</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="pixel_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>107</width>
     <height>20</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>pixel:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>20</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Full Image</string>
   </property>
  </widget>
  <widget class="QLabel" name="mean_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>107</width>
     <height>20</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>mean: </string>
   </property>
  </widget>
  <widget class="QLabel" name="median_label">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>40</y>
     <width>107</width>
     <height>20</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>median: </string>
   </property>
  </widget>
  <widget class="QLabel" name="sd_label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>107</width>
     <height>20</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>SD: </string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QGraphicsViewCustom</class>
   <extends>QGraphicsView</extends>
   <header location="global">qgraphicsviewcustom.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
